src/main/java/net/dv8tion/jda/api/entities/MessageReaction.java

```diff
@@ -184,6 +184,31 @@ public TextChannel getTextChannel()
         return getChannel() instanceof TextChannel ? (TextChannel) getChannel() : null;
     }
 
+    /**
+     * The {@link net.dv8tion.jda.api.entities.VoiceChannel VoiceChannel} this Reaction was used in
+     * or {@code null} if this is not from type {@link net.dv8tion.jda.api.entities.ChannelType#VOICE ChannelType.VOICE}!
+     *
+     * @return The {@link net.dv8tion.jda.api.entities.VoiceChannel VoiceChannel} or {@code null}
+     */
+    @Nullable
+    public VoiceChannel getVoiceChannel()
+    {
+        return getChannel() instanceof VoiceChannel ? (VoiceChannel) getChannel() : null;
```

Comment:
I know that this is the functionality that already exists, but we should probably change this system to throw `IllegalStateException` instead of returning `null`.